include/rcpputils/join.hpp

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2019, Open Source Robotics Foundation, Inc.
+// All rights reserved.
+//
+// Software License Agreement (BSD License 2.0)
```

Comment:
Was this copied from somewhere? Why the BSD License and not Apache 2?

Reply:
Oops, that's a bad copy-paste.

Reply:
Back to Apache 2 in https://github.com/ros2/rcpputils/pull/20/commits/1aceb4f6547fc3c6db95e7e63cf57b0006e92aa2.